api-reference/echo-node-api/network-broadcast-api.md

```diff
@@ -1,58 +1,74 @@
 # Network broadcast API
 
-#### broadcast_transaction(signed_transaction)
+### broadcast_transaction(trx)
 
 Broadcast a transaction to the network.
 
 The transaction will be checked for validity in the local database prior to broadcasting. If it fails to apply locally, an error will be thrown and the transaction will not be broadcast.
 
+#### Parameters
+
+| Option                          | Description                  |
+|---------------------------------|:-----------------------------|
+| `const signed_transaction &trx` | The transaction to broadcast |
```

Comment:
Invalid type of options. Should be human readable

The same in the next methods

Reply:
Fixed.